app/src/main/res/values/colors.xml

```diff
@@ -14,13 +14,21 @@
 
     // --- Pastels ---//
 
-    <color name="light_mint">#a8e6cf</color>
+
     <color name="light_yellow">#dcedc1</color>
     <color name="light_orange">#ffd3b6</color>
     <color name="light_red_orange">#ffaaa5</color>
     <color name="light_red">#ff8b94</color>
 
 
+    //--- Purple palette ---//
+    <color name="purple">#5d54a4</color>
+    <color name="light_purple">#9d65c9</color>
+    <color name="purp_palette_pink">#d789d7</color>
+    <color name="purp_palette_indigo">#2a3d66</color>
+
+
+
```

Comment:
this is just a thought but I wonder if we could name these colors better? maybe in addition to the color we can add what it's used for? If it's broad enough. for example instead of `light_purple` we write `positive_buttons_purple` or `bottom_nav_icons_purple`. the name `purp_palette_pink` is hard to remember.  I know it's a button so if i search for "button" i'll see `positive_buttons_purple` pop up. know what I mean? But again just a thought, idk what the rules are here, I have to refresh myself on the google style docs

Reply:
yea we can do that, it would have to be a bit universal since we could use the same color for text's, buttons, backgrounds, and so on. I tried to make it specific but general but I'm def down to make it easier to search